lib_jobs_rs/src/keyword_field.rs

```diff
@@ -0,0 +1,50 @@
+use std::ffi::{CStr, CString};
+
+/// Make a keyword suitable for comparing to other keywords
+///
+/// # Examples
+/// ```
+/// use std::ffi::{CStr, CString};
+///
+/// let original = CString::new("My title /").unwrap().into_raw();
+/// let result = unsafe { lib_jobs::normalize_keyword(original) };
+/// assert_eq!(unsafe { CStr::from_ptr(result) }.to_str().unwrap(), "My title ");
+/// ```
+///
+/// # Safety
+///
+/// Since this function reads arbitrary memory from a C pointer,
+/// there are some safety considerations described in
+/// <https://doc.rust-lang.org/std/ffi/struct.CStr.html#method.from_ptr>
+#[no_mangle]
+pub unsafe extern "C" fn normalize_keyword(raw_string_ptr: *const i8) -> *const i8 {
```

Comment:
This function is mainly glue code: taking the pointer that ruby provided, calling the logic of our function, and then returning a new pointer to the corrected string.

Reply:
I don't think we need the unsafe keyword in the function signature since the only unsafe call is wrapped in an unsafe block. We don't need to use an unsafe block in the test if this function isn't marked as unsafe.